include/TypesafeCollection.h

```diff
@@ -48,7 +48,7 @@ class TypesafeCollection
 	}
 
 	virtual ~TypesafeCollection(){}
-	TypesafeCollection<T>(const TypesafeCollection<T>&) = delete;
+	TypesafeCollection(const TypesafeCollection<T>&) = delete;
 	TypesafeCollection<T>& operator=(const TypesafeCollection<T>&) = delete;
```

Comment:
```suggestion
	TypesafeCollection& operator=(const TypesafeCollection<T>&) = delete;
```
I suppose since it's also necessary for the copy constructor.

Reply:
It's not necessary, it compiles fine with `<T>` with both gcc 11 and 13, that's why I didn't change it and in the other places

Reply:
Interesting. Would have expected this to be symmetric.